src/gui/mousecursor.cpp

```diff
@@ -53,6 +54,11 @@ void MouseCursor::set_mid(int x, int y)
   m_mid_y = y;
 }
 
+void MouseCursor::set_icon(SurfacePtr icon_)
```

Comment:
This doesn't work, because set_icon(nullptr). SurfacePtr is std::shared_ptr<Surface>, so it is already a pointer.